### PR TITLE
Allow populating config with env vars

### PR DESCRIPTION
**Pull request by yairsimantov20**
# Description

What - We need to enable a way to pass the config with env vars
How - users can pass the env vars in a SNAKE_CASE formatting and __ for a nested key

## Type of change

- [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: .github/workflows/release-integrations.yml [hunk 1]
```diff
@@ -4,6 +4,11 @@ on:
     branches:
       - main
   workflow_dispatch:
+    inputs:
+      LATEST:
```
Comment (dvirsegev) [inline .github/workflows/release-integrations.yml:8]: should be always the latest no?

File: integrations/gitlab/pyproject.toml [hunk 2]
```diff
@@ -10,7 +10,7 @@ aiofiles = "^0.6.0"
 python-gitlab = "^3.14.0"
 pathlib = "^1.0.1"
 jsonschema = "^4.17.3"
-port-ocean = {version = "0.1.1", extras = ["cli"]}
+port-ocean = {version = "0.1.2rc3", extras = ["cli"]}
```
Comment (dvirsegev) [inline integrations/gitlab/pyproject.toml:13]: are you sure?

**Review by dvirsegev — APPROVED** [review]
LGTM

File: .github/workflows/release-integrations.yml (hunk 1 above)
Comment (yairsimantov20) [reply .github/workflows/release-integrations.yml:8]: when im building manually i want the option to build images for development

File: integrations/gitlab/pyproject.toml (hunk 2 above)
Comment (yairsimantov20) [reply integrations/gitlab/pyproject.toml:13]: will be change after this pr
i need to publish 0.1.2

**Pull request merged**